python/ctsm/crop_calendars/interpolate_gdds.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
This is a new script to interpolate GDD's. Currently it's only used from within the RXCROPMATURITY system tests. Might there be a reason a user would want to do this outside of the system testing? If so having a skeleton program in tools/crop_calendars might make sense.

Reply:
I don't think so; it's a pretty narrow use case.